components/ColonyQuestions.js

```diff
@@ -65,12 +60,10 @@ export default function ColonyQuestions({ currentColony }) {
             onChange={handleInputChange}
           />
         </div>
-      </StyledQuestionCard>
+      </StyledCard>
 
-      <StyledQuestionCard>
-        <StyledQuestionLabel htmlFor="question2">
-          2. Königin?
-        </StyledQuestionLabel>
+      <StyledCard>
+        <StyledLabel htmlFor="question2">2. Königin?</StyledLabel>
```

Comment:
The same goes for this label; the problem is that you use a label, but you don't have a corresponding input field.